.travis.yml

```diff
@@ -23,7 +24,32 @@ matrix:
     - rust: nightly
       env: FEATURES=""
   allow_failures:
-    - rust: nightly
+  - rust: nightly
 
 script:
   - cargo test --no-default-features --features "$FEATURES"
+
+before_deploy:
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    make dmg;
+    export DEPLOY_FILE="$PWD/target/release/osx/Alacritty.dmg";
+  fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    cargo build --release;
+    cargo install cargo-deb;
+    export DEPLOY_FILE=$( cargo deb --no-build );
+  fi
+- echo "DEPLOYING FILE $DEPLOY_FILE";
+
+deploy:
+  provider: releases
+  api_key:
+    secure: o2Q39A2JAKeMEYD0dFJTpYJMpAsHYcUa4DIXApqn/ygXQzcfPpnLelCWb70zOlICIk3ICETlzeLfF7s/q802jy5aEFLbbmIp+rT0SWlRiShuQqLPkwlK7Cqh2nUAVPoozYn1y9ha8kozOL3xgDxI9RGHMC+7ZV9duVLWUqz5Ah7qBoOx9QemDMrdPJ7zi8vFvJp4oW3EV7wQRRfEmFkqOAxbAWhLeTNGVR9ECjUZIgXWRz1kjqVz6dJrLauH/D068RSV/7JM1dlzrMkBsmRe8xVOTnAiHIoGMB4UaUTLz8hJNH+KUmXfprfBKuFEQ7jDnYq+6/y200VYuRfRsU2EOqYjzHXmWD6E3vkHhVdrUUfFmG9tfAzIXFZxs0LVIszWgqqMypmuLNxgYcfpMmo+YeHGWX59HOVwXmvY38ix1++zLh44kpRuX0mYFuPdH6gRdeKPr7jyueODKRwsegh1d8rVl7T0HQpBmDcWHBidOf8Aw8NTrsEoRQFlS/eVZyGnZet/nf6Otenp/5MIBA3BHO9R9QaCo0FT3Yi7FL/kkoEZs+lN5mKJulZ6hpmG760h+mYBqw7buLKcCR3qBiuXOpgfzNWgOoAGb3G7eUAhCf1zHTEcdL6U/SEA2ofpaPZGqrcuZ7aTgnJ6lWU4YWI8X+hAouv6IOk/UoxiONIUwHE=
+  skip_cleanup: true
+  overwrite: true
```

Comment:
I think it would be better to remove this line. Creating a build manually when the process fails and overwriting existing releases is no problem. However once a file is overwritten it can't be undone.

Reply:
:+1: